core/renderers/common/info.js

```diff
@@ -191,9 +191,9 @@ Blockly.blockRendering.RenderInfo.prototype.getRenderer = function() {
 Blockly.blockRendering.RenderInfo.prototype.measure = function() {
   this.createRows_();
   this.addElemSpacing_();
-  this.addRowSpacing_();
   this.computeBounds_();
   this.alignRowElements_();
+  this.addRowSpacing_();
```

Comment:
This was moved up deliberately before computeBounds, what's the reasoning to move it back down?